src/sentry/static/sentry/less/base.less

```diff
@@ -195,18 +189,6 @@ img {
   max-width: 100%;
 }
 
-/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
```

Comment:
this might still apply?

Reply:
Yeah we need to keep this part of the comment.